sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -869,15 +869,18 @@ lsc_gather(AccessorTy acc, __ESIMD_NS::simd<uint32_t, N> offsets,
 /// for the data size are \c lsc_data_size::u32, \c lsc_data_size::u64,
 /// \c lsc_data_size::u8, \c lsc_data_size::u16.
 /// When data size is either  \c lsc_data_size::u8 or \c lsc_data_size::u16
-/// the data is treated as 32 bit data. Allowed NElts values for 64 bit
-/// data are 1, 2, 3, 4, 8, 16, 32, 64.
+/// the data is treated as 32 bit data. Allowed \c NElts values for 64 bit
```

Comment:
nit: it is easier to read if "Allowed \c NElts..." if it starts from the new line.